pages/facts.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <link rel="stylesheet" href="../stylesheets/facts.css">
   <title>Document</title>
 </head>
 <body>
+    <header>
+    <h1>Fun Facts</h1>
+    </header>
+    <section class="container">
+        <div>
+            <h3>Indoor plants improve air quality</h3>
```

Comment:
In the MDN docs it's highly suggested to not skip headings:
> A common navigation technique for users of screen reading software is jumping from heading to quickly determine the content of the page. Because of this, it is important to not skip one or more heading levels. Doing so may create confusion, as the person navigating this way may be left wondering where the missing heading is.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#navigation

Here the suggestion would be to use `h2` instead of the `h3`s and style the `h3`s as needed to match the smaller font size desired.